### PR TITLE
prevent crash on WC sessions updates

### PR DESCRIPTION
**Pull request by estebanmino**
- Added optional chaining
- Remove and disconnect connectors in bad state, no `peerMeta`

https://sentry.io/organizations/rainbow-me/issues/2672039238/events/a3ca4ad08fcb4b40b385b470d413a6c4/?project=1855565&query=is%3Aunresolved+level%3Afatal+user.ip%3A174.241.64.253&statsPeriod=14d

### CODE REVIEW

File: src/redux/walletconnect.js [hunk 1]
```diff
@@ -601,7 +601,10 @@ export const walletConnectDisconnectAllByDappUrl = dappUrl => async (
 ) => {
   const { walletConnectors } = getState().walletconnect;
   const matchingWalletConnectors = values(
-    pickBy(walletConnectors, connector => connector.peerMeta.url === dappUrl)
+    pickBy(
+      walletConnectors,
+      connector => connector?.peerMeta?.url === dappUrl || !connector?.peerMeta
```
Comment (estebanmino) [inline src/redux/walletconnect.js:606]: added `!connector?.peerMeta` as a condition to remove the connector in bad state

**Review by skylarbarrera — APPROVED** [review]
we love ❔

**Review by brunobar79 — APPROVED** [review]
🚀

**Comment by skylarbarrera** [comment]
merging so i can finally do a release tonight 💪

**Pull request merged**